.github/CONTRIBUTING.md

```diff
@@ -83,17 +72,62 @@ yarn link webpack-cli
   yarn build
   ```
 
+
+## Testing
+
+### Using npm
+
 * Run all the tests with:
-  - `yarn test`
+  
+  ```bash
+  npm run test
+  ```
 
-* To test a single CLI (flag) test case:
-    - `BIN_TEST_CASES_GREP=/myCase jest test/BinTestCases.test.js`
+* Test a single CLI test case:
+  
+  ```bash
+  npx jest path/to/my-test.js 
+  ```
 
-* To test a single CLI (other type of) test case:
-    - `yarn jest path/to/my-test.js`
+* You can also install jest globally and run tests without npx:
 
-* To test linting:
-    - `yarn lint && yarn tslint`
+  ```bash
+  npm i -g jest
+  jest path/to/my-test.js
+  ```
+
+* You can run the linters:
+
+  ```bash
+  npm run lint && npm run tslint
+  ```
+
+### Using yarn
+
+* Run all the tests with:
+  
+  ```bash
+  yarn test
+  ```
+
+* Test a single CLI test case:
+  
+  ```bash
+  npx jest path/to/my-test.js
```

Comment:
With yarn you don't need npx.

You can run 'yarn jest' and it will use the local version installed

Reply:
Oh great, nice to know!